app/code/community/SomethingDigital/InfluxDb/Model/Measurement/Changelog.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Sends data about status of Enterprise_Index changelogs
+ *
+ * measurement: changelog
+ * tag(s): name
+ * field(s): metadata_version, version
+ *
+ */
+class SomethingDigital_InfluxDb_Model_Measurement_Changelog
+    extends SomethingDigital_InfluxDb_Model_Measurement_Abstract
+    implements SomethingDigital_InfluxDb_Model_MeasurementInterface
+{
+    public function send()
+    {
+        $metadatas = Mage::getModel('enterprise_mview/metadata')->getCollection();
+        $data = array();
+        foreach ($metadatas as $metadata) {
+            $data[] = $this->line($metadata);
+        }
+
+        $this->api->write(implode(PHP_EOL, $data));
+    }
+
+    protected function line($metadata)
+    {
+        $changelog = Mage::getModel('enterprise_index/changelog', array(
+            'connection' => $this->readCon,
+            'metadata' => $metadata,
+        ));
+
+        return 'changelog,name=' . $metadata->getChangelogName() .
```

Comment:
A lot of the measurements are using string concatenation.

https://github.com/sdinteractive/SomethingDigital_InfluxDb/blob/develop/app/code/community/SomethingDigital/InfluxDb/Model/Measurement/Cron.php#L54-L58

It is a bit ugly, though.